### PR TITLE
Fix/doc improvement tools debugging

### PR DESCRIPTION
**Pull request by pri-lk**
┆Issue is synchronized with this [Wrike task](https://www.wrike.com/open.htm?id=982212872)

### CODE REVIEW

File: hugo/content/3_how-tos/35_tooling_and_debugging/355_record-rosbags-for-packages.md [hunk 1]
```diff
@@ -22,61 +22,41 @@ tags:
...
-    <li>**Upload Rate**: Specify the upload rate for the ROS bag file to be uploaded in the rapyuta.io platform. </li>
-    <li>**Rate Units**: Specify the unit for the upload rate from the drop-down menu. You can select **Bytes/s**, **KB/s**, or**MB/s**.  </li>
-    <li>**Purge After**: If you want to delete the ROS bag file after it has been successfully uploaded to the rapyuta.io platform, click the toggle-button.</li>
-    </ul> 
-![device-rosbags](/images/dev-guide/rosbag-jobs/advanced-rosbags-device.png?classes=border,shadow&width=35pc)
-
-f. To add the ROS bag job and start recording the defined topics, click **Add**.
-
+{{% /notice %}}|
+| Upload Rate | (Applicable only for device runtime) Specify the upload rate for the ROS bag file to be uploaded in the rapyuta.io platform.|
```
Comment (shiv-mohith) [inline hugo/content/3_how-tos/35_tooling_and_debugging/355_record-rosbags-for-packages.md:54]: Does this make sense?
```suggestion
| Upload Rate (Applicable only for device runtime) | Specify the upload rate for the ROS bag file to be uploaded in the rapyuta.io platform.|
```

File: hugo/content/3_how-tos/35_tooling_and_debugging/357_using-rapyuta-io-python-sdk.md [hunk 2]
```diff
@@ -22,34 +22,22 @@ tags:
...
-services in your python applications.
-
-
+rapyuta.io Python SDK enables you to provision packages either on the cloud or on a device. Add a dependent deployment, and access various rapyuta.io resources and services in your python applications.
 
 ## Installation
 
-It is recommended you install the latest Python SDK
-using **pip**(the most popular tool for installing python packages).
+It is recommended you install the latest Python SDK using **pip**(the most popular tool for installing python packages).
```
Comment (shiv-mohith) [inline hugo/content/3_how-tos/35_tooling_and_debugging/357_using-rapyuta-io-python-sdk.md:29]:
```suggestion
It is recommended you install the latest Python SDK using **pip** (the most popular tool for installing python packages).
```

File: hugo/content/3_how-tos/35_tooling_and_debugging/354_view-deployment-logs.md [hunk 3]
```diff
@@ -46,76 +40,35 @@ cloud runtime. It will publish the ROS topic *_/telemetry_* with **Maximum** **Q
...
 Any errors or exceptions that occur while logs are being generated are redirected to **stderr** output window, but otherwise,
 a valid logs output is displayed in **stdout** window.
 
-![stdout or stderr](/images/core-concepts/logging/deployment-logs/historical-logs/stdout-stderr.png?classes=border,shadow&width=50pc)
-
-
-
 ### Indexed Logs
 
 To view the corresponding deployment logs in real-time, click **Live Logs** tab. They are streamed in a terminal-like window and can be useful to view and debug your application in the cloud as it generates new logs. The rapyuta.io platform allows you to view the live
```
[the remark this answers is not in the document]
Comment (pri-lk) [reply hugo/content/3_how-tos/35_tooling_and_debugging/354_view-deployment-logs.md]: @Shiv, is it "The rapyuta.io platform allows you to view the live logs"? 
The existing content is incomplete as well.

File: hugo/content/3_how-tos/35_tooling_and_debugging/355_record-rosbags-for-packages.md (hunk 1 above)
Comment (pri-lk) [reply hugo/content/3_how-tos/35_tooling_and_debugging/355_record-rosbags-for-packages.md:53]: It's better to have it under description

File: hugo/content/3_how-tos/35_tooling_and_debugging/354_view-deployment-logs.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (shiv-mohith) [reply hugo/content/3_how-tos/35_tooling_and_debugging/354_view-deployment-logs.md]: I feel this sentence itself is not necessary

File: hugo/content/3_how-tos/35_tooling_and_debugging/355_record-rosbags-for-packages.md (hunk 1 above)
Comment (shiv-mohith) [reply hugo/content/3_how-tos/35_tooling_and_debugging/355_record-rosbags-for-packages.md:53]: okay

File: hugo/content/3_how-tos/35_tooling_and_debugging/357_using-rapyuta-io-python-sdk.md (hunk 2 above)
Comment (shiv-mohith) [reply hugo/content/3_how-tos/35_tooling_and_debugging/357_using-rapyuta-io-python-sdk.md:29]: what about this?

**Review by shiv-mohith — APPROVED** [review]

**Pull request merged**